frontend/src/app/modules/header/header-item/header-item.component.sass

```diff
@@ -61,24 +62,24 @@ header
 
       text-decoration: none
       border-radius: 3px
-    
+
     .tab a:hover
-     background-color: $athens-gray
-     border: 5px solid $athens-gray
-     cursor: pointer
+      background-color: $athens-gray
```

Comment:
In this file, you have so small indent
You need to follow our common code style

Reply:
Tab as indent?